packages/_metapackage/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@jupyterlite/metapackage",
+  "private": true,
+  "version": "0.1.7",
```

Comment:
Should we stick to `0.1.0` for now until we start publishing packages?

Reply:
sure, sorry, went off `application`. it's private, so it doesn't matter... but think it has to have _something_ for... reasons.

Reply:
bde8c6d

Reply:
:+1: